test/unit/test_Rewriting.cc

```diff
@@ -1,18 +1,22 @@
-//
-// Created by Martin Blicha on 17.05.20.
-//
+/*
+ * Copyright (c) 2020-2024 Martin Blicha <martin.blicha@gmail.com>
+ *
+ *  SPDX-License-Identifier: MIT
+ *
+ */
 
 #include <gtest/gtest.h>
-#include <ArithLogic.h>
-#include <BoolRewriting.h>
-#include <Logic.h>
-#include <ArithmeticEqualityRewriter.h>
-#include <DivModRewriter.h>
-#include <MainSolver.h>
-#include <DistinctRewriter.h>
+
+#include "ArithLogic.h"
+#include "BoolRewriting.h"
```

Comment:
I am not sure what the difference between `src/rewriters` and `src/simplifiers` is.
Also, to maintain consistency, why this is called `*Rewriting.h` and the new file `Rewritings.h`?

Reply:
#740